deepmd/utils/data_system.py

```diff
@@ -560,11 +560,11 @@ def print_summary(self, name):
         log.info("found %d system(s):" % self.nsystems)
         log.info(
             ("%s  " % self._format_name_length("system", sys_width))
-            + ("%6s  %6s  %6s  %5s  %3s" % ("natoms", "bch_sz", "n_bch", "prob", "pbc"))
+            + ("%6s  %6s  %6s  %9s  %3s" % ("natoms", "bch_sz", "n_bch", "prob", "pbc"))
         )
         for ii in range(self.nsystems):
             log.info(
-                "%s  %6d  %6d  %6d  %5.3f  %3s"
+                "%s  %6d  %6d  %6d  %.3e  %3s"
```

Comment:
```suggestion
                "%s  %6d  %6d  %6d  %9.3e  %3s"
```
Although them seem to be equivalent when printing a probability